internal/pkg/fleetapi/action.go

```diff
@@ -542,12 +549,14 @@ func (a *Actions) UnmarshalJSON(data []byte) error {
 				ActionType: response.ActionType,
 			}
 		case ActionTypeInputAction:
+			// Only INPUT_ACTION type actions could possibly be signed as of March 1, 2022
```

Comment:
```suggestion
			// Only INPUT_ACTION type actions could possibly be signed as of https://github.com/elastic/elastic-agent/pull/2348
```

Just linking back to the PR seems more helpful.